### PR TITLE
Add Userscript installation method

### PR DESCRIPTION
**Pull request by mintexists**
I am using `@require` as a way to just load the script from the url

### CODE REVIEW

**Comment by DavidBuchanan314** [comment]
Hm, what are the pros and cons of using jsdelivr? I'd rather not add a supply-chain dependency without good reason

**Review by DavidBuchanan314 — COMMENTED** [review]
This URL is specific to your fork - I think it should be possible to use a relative URL link, then it'll work for both.

File: README.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ Go to `about:addons` and click the settings icon. Select "debug addons".
 
 Click "Load temporary add-on", and select the manifest file.
 
+## Installation (Userscript - tested with Violentmonkey)
+
+Visit [This Link](https://github.com/enbyautumn/TwitterHD/raw/main/twitterhd.user.js) to automatically install
```
Comment (DavidBuchanan314) [inline README.md]: (this URL, that is) (I'm not very good at using github's code review interface heh)

**Comment by mintexists** [comment]
> Hm, what are the pros and cons of using jsdelivr? I'd rather not add a supply-chain dependency without good reason

I had some vague memories over GitHub not liking it when people use raw file things, figured it’d be better safe than sorry 

I’ll make that install link relative, wasn’t thinking when I used the non-relative link lol

**Comment by DavidBuchanan314** [comment]
I think I'd rather not have the jsdeliver link for now - if it becomes a problem in the future than I'll reconsider. Out of principle, I'd rather have potential availability issues than potential integrity issues.

**Comment by mintexists** [comment]
linking to the raw file relatively seems not possible, ill just link directly to the raw on your branch

**Comment by mintexists** [comment]
Should be all good to go

**Comment by DavidBuchanan314** [comment]
Thanks :)

**Comment by mintexists** [comment]
oh shoot i messed up a link

**Comment by mintexists** [comment]
is it possible to reopen the pull request? if not u could edit it manually
```
Visit [This Link](https://raw.githubusercontent.com/DavidBuchanan314/TwitterHD/main/twitterhd.js) to automatically install
Visit [This Link](https://raw.githubusercontent.com/DavidBuchanan314/TwitterHD/main/twitterhd.user.js) to automatically install
```

**Comment by DavidBuchanan314** [comment]
Already fixed that :)

**Pull request merged**